api/api.go

```diff
@@ -259,6 +259,10 @@ func (api *API) listAlerts(w http.ResponseWriter, r *http.Request) {
 		res []*APIAlert
 	)
 
+	// Initialize result slice to prevent api from returning `null` when there
+	// are no alerts present
+	res = []*APIAlert{}
```

Comment:
You can move this into the variabel declaration block above. The matchers too actually:

```go
var (
    err error
    res = []*APIAlert{}
    matchers = []*labels.Matcher{}
)
```